content/en/docs/handling-errors.md

```diff
@@ -54,8 +55,8 @@ other interfaces to get more information about an error.
 ## Specific Error Interfaces
 
 Other packages might provide their own error interfaces. For example,
-the {{< apiref "feature/s3/manager" >}}feature/s3/manager{{< /apiref >}} package
-provides a {{< apiref "feature/s3/manager#MultiUploadFailure" >}}MultiUploadFailure{{< /apiref >}}
+the [feature/s3/manager]({{< apiref "feature/s3/manager" >}}) package
+provides a [MultiUploadFailure]({{< apiref "feature/s3/manager#MultiUploadFailure" >}})
 interface to retrieve the upload ID. This is helpful when you must
 manually clean up a failed multi-part upload.
 
```

Comment:
This section should use `errors.As` instead of type assertions.

Reply:
This content hasn't been rewritten yet, please review the content changes under `content/en/docs/configuring-sdk/*`